packages/math-input/src/components/input/math-input.tsx

```diff
@@ -908,11 +908,21 @@ class MathInput extends React.Component<Props, State> {
                 overrides.css. */}
                 <div
                     className="keypad-input"
-                    // @ts-expect-error - TS2322 - Type 'string' is not assignable to type 'number | undefined'.
-                    tabIndex={"0"}
+                    tabIndex={0}
                     ref={(node) => {
                         this.inputRef = node;
                     }}
+                    onFocus={() => {
+                        this.focus();
+                    }}
+                    onBlur={() => {
+                        this._hideCursorHandle();
+                        this.mathField.blur();
+                        this.setState({
+                            focused: false,
+                            handle: {visible: false},
+                        });
```

Comment:
I think we'll just want to call `this.blur()` here right?
```suggestion
                        this.blur();
```

Reply:
Calling this.blur actually causes us to call our parent methods, which causes the keypad to close undesirably:

` this.props.onBlur && this.props.onBlur();`

However I could make this.blur() take a boolean that conditionally runs this line if that's preferrable :)